### DISCUSSION TITLE
binary ninja start on Ubuntu 24.04 error( libEGL.so.1 not found)

### DISCUSSION THREAD

**Discussion by csb1109** [Vector35/binaryninja-api#5917] [Q&A] [answered]
I install binaryninja with command "./binaryninja/scripts/linux-setup.sh" and install success.
when I move binaryninja directory and start binaryninja with command "./binaryninja" then "libEGL.so.1 not found" error occur.
For explanation, refer to the screenshot
![image](https://github.com/user-attachments/assets/c05caba0-deef-4416-9cc1-290058c056ec)
Please someone help me . . .

**Comment by xusheng6** [comment]
> I install binaryninja with command "./binaryninja/scripts/linux-setup.sh" and install success. when I move binaryninja directory and start binaryninja with command "./binaryninja" then "libEGL.so.1 not found" error occur. For explanation, refer to the screenshot ![image](https://private-user-images.githubusercontent.com/62738644/366068135-c05caba0-deef-4416-9cc1-290058c056ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5NzgzNDQsIm5iZiI6MTcyNTk3ODA0NCwicGF0aCI6Ii82MjczODY0NC8zNjYwNjgxMzUtYzA1Y2FiYTAtZGVlZi00NDE2LTljYzEtMjkwMDU4YzA1NmVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEwVDE0MjA0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwYjBmZWMyYTM1OGUyZDVjMmMxNWExYmUxM2EwZDljNGRmNDk4MGFkZjRkOTVlZjdlN2NkMjVjYjJjMmUyOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jNYcHyoEX3mhgJkE_MvMIFmbqiVhyiW8pq5R8UmAcZg) Please someone help me . . .

Hi, @GitBasicUser, sorry for the inconvenience. Which specific version Binary Ninja are you trying to run?

**Reply by csb1109** [reply]
How can I see the version of Binary Ninja??
I purchase license on last friday, so I guess I have latest version

**Comment by xusheng6** [comment]
`libEGL` is not a file that we ship, so you will probably need to figure it out on your end. Try installing `libegl1` with `apt install libegl1` may fix the issue. Or search relevant ones with `apt search libegl`. However, I am not super familiar with the package so you need to be careful with it.

Do you machine has a NVIDIA graphics card?

**Reply by csb1109** [reply]
Oh, I try command "apt install libegl1" and libEGL error fixed!! Thanks for your help!!
but second error occur. . . that "libxkbcommon.so.0"(screenshot)

and my machine has A5000 GPU on graphics card line second screenshot.

![1](https://github.com/user-attachments/assets/3a240750-91ae-4cdc-b2f5-ca0a12f0a64b)
![화면 캡처 2024-09-11 144831](https://github.com/user-attachments/assets/57dbc2c5-b380-4c52-aeb6-784d9dbd69e0)

**Reply by csb1109** [reply]
Oh I solved second error but finally I encounter "xcb error" like screenshot.
Can you help me to solve this problem please..??
![image](https://github.com/user-attachments/assets/ecd03fba-425e-4314-90fc-a9266b80318d)

**Reply by xusheng6** [reply]
yeah, that `xcb` looks familiar to me, let me see if I can find anything

**Reply by negasora** [reply, marked as answer]
You probably need to `apt install libxcb-cursor0`
